bundles/GraphicServiceOrder/Controller/GraphicServiceOrderController.php

```diff
@@ -32,12 +32,20 @@ class GraphicServiceOrderController extends AbstractController
      *
      * @return \Symfony\Component\HttpFoundation\RedirectResponse|\Symfony\Component\HttpFoundation\Response
      *
+     * @throws \GuzzleHttp\Exception\GuzzleException
      * @throws \Twig\Error\LoaderError
      * @throws \Twig\Error\RuntimeError
      * @throws \Twig\Error\SyntaxError
      */
     public function newOrder(Request $request, OrderService $orderService, TokenStorageInterface $tokenStorage)
     {
+        $user_email = $tokenStorage->getToken()->getUser()->getEmail();
```

Comment:
CamelCase instead of snake_case for variable names

Reply:
Fixed